ddl/000008_export_cfg.down.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE catalog_branches_export;
```

Comment:
IF EXISTS

Reply:
I see we do that everywhere, so did it here too.  Not sure it makes anything safer.